### DISCUSSION TITLE
Make planning public again

### DISCUSSION THREAD

**Discussion by jimmykarily** [kairos-io/kairos#2883] [General]
We used to have a public link for our planning meetings. At some point we had technical issues with the meeting room (was it jitsi?) and we switched to doing it over Slack calls. With our community growing, we should considering switching back to open planning calls so others can participate.

This will allow people to express their concerns, priorities and insights allowing Kairos to serve its community better.

**Comment by mauromorales** [comment]
I like the idea, and I'd add the following:

- recordings because it will be impossible to have a time and day that will be good for everyone across the world
- have a space at the beginning or end of the call for people to raise their concerns in person, and a public document (maybe on GitHub?) for those who want to do this async

WDYT?

**Reply by jimmykarily** [reply]
regarding the second one (first half of it), if they are participating in the planning in person, do we need a specific slot to raise concerns? I mean, they might raise concerns at any time during the call, no?

**Reply by jimmykarily** [reply]
I agree with the public document. I agree with the recording although this will either force us to use a tool that does it automatically (google things...?) or record/process/upload manually which is time consuming. But we'll figure something out.

**Reply by anthonyra** [reply]
With regards to recording - might make sense to leverage something that works well with jitsi? 
https://jitsi.support/jibri-recording-with-scaling

Using a Github Discussion as the public document for questions/agenda might make sense also? Not sure how current planning meetings are structured but having a specific time slot in the meeting may make sense to not derail the entire meeting if a topic lasts a while.

Here's an example of a Github issue being used for the public document - https://github.com/keylime/meetings/issues/81

**Comment by jimmykarily** [comment]
Calendar is now updated with the planning meething and link: https://calendar.google.com/calendar/u/0/embed?src=c_6d65f26502a5a67c9570bb4c16b622e38d609430bce6ce7fc1d8064f2df09c11@group.calendar.google.com&ctz=Europe/Rome

**Comment by jimmykarily** [comment]
Regarding agenda, I would stick to the current format (we discuss over the project board: https://github.com/orgs/kairos-io/projects/1/views/1)  until it proves insufficient.

**Comment by jimmykarily** [comment]
We got @bencorrado in the planning call today but the time is really bad for his timezone. Let's find a better slot that works for at least the people we know are joining. Here are some suggestions (please correct me if I got your timezone wrong):

Current time: 8a.m. UTC which translates to:
   11:00 @jimmykarily 
   10:00 @mudler , @Itxaka , @mauromorales 
   01:00 @bencorrado
   02:00 @anthonyra

Suggested slots:

- 16:00 UTC which translates to:
   19:00 @jimmykarily 
   18:00 @mudler , @Itxaka , @mauromorales 
   09:00 @bencorrado
   10:00 @anthonyra

- 17:00 UTC which translates to:
   20:00 @jimmykarily 
   19:00 @mudler , @Itxaka , @mauromorales 
   10:00 @bencorrado
   11:00 @anthonyra

- 18:00 UTC which translates to:
   21:00 @jimmykarily 
   20:00 @mudler , @Itxaka , @mauromorales 
   11:00 @bencorrado
   12:00 @anthonyra


If anyone wants to join that I didn't include in the list, please chime in. If you want to suggest another slot, feel free to do so. Ben and Anthony if you are early birds let us know, maybe we can try some earlier slots.

**Reply by mauromorales** [reply]
Thanks for doing all the hours @jimmykarily, and thanks @bencorrado for joining our planning at those ungodly hours!

From my side 16:00 and 17:00 UTC would be tricky most days, mainly because it's dinner time or the time people can easily meet after work here e.g. Dr visit. So I'd prefer not

and for 18:00 UTC, I think that would be ok most of the time, but sometimes I'd need to skip

Since I'm also not 100% on the backlog, I think it would be ok for me to miss sometimes or even often, so don't be blocked to make a change because of me. If you decide to go with this option, I'd appreciate it if we can record it or have some way to share "highlights"

**Reply by anthonyra** [reply]
@bencorrado and I are working together - If one of us is there we can sync up later. 

The earliest for me would be 11:00 UTC - 14:00 UTC and 14:00 UTC being the earliest for Ben. I think it'd make more sense to get it earlier for the core team vs later at night for personal agendas.

**Reply by bencorrado** [reply]
14:30UTC or later works best for me. So beyond that pick the best time for you guys and we will figure it out.

Thank you all for working to accommodate us!

**Reply by jimmykarily** [reply]
ok this looks like the best slot then:

14:30 UTC which translates to:
17:30 @jimmykarily
16:30 @mudler , @Itxaka , @mauromorales
07:30 @bencorrado
08:30 @anthonyra

I'll move the calendar event

**Reply by Itxaka** [reply]
> ok this looks like the best slot then:
> 
> 14:30 UTC which translates to:
> 17:30 @jimmykarily
> 16:30 @mudler , @Itxaka , @mauromorales
> 07:30 @bencorrado
> 08:30 @anthonyra
> 
> I'll move the calendar event

I can only do 15 minutes of planning at that point as I have to pick up my 3 children
